AppCenterData/AppCenterData.xcodeproj/project.pbxproj

```diff
@@ -856,6 +860,7 @@
 			developmentRegion = English;
 			hasScannedForEncodings = 0;
 			knownRegions = (
+				English,
```

Comment:
What's this change?

Reply:
Probably introduced by Xcode itself.

Reply:
I removed that. I wonder which version of Xcode was used for this PR.